src/FCM.js

```diff
@@ -195,9 +195,11 @@ function _APNSToFCMPayload(requestData) {
         // compatible with how the APNS.js + node-apn work
         apnsPayload['apns']['payload']['aps']['alert'] = coreData.alert;
       } else {
-        // When we receive a value, prepare `alert` dictionary
+        // When we receive a value, prepare `alert` dictionary if needed
         // and set its `body` property
-        apnsPayload['apns']['payload']['aps']['alert'] = {};
+        if (!apnsPayload['apns']['payload']['aps'].hasOwnProperty('alert')) {
```

Comment:
I understand that this is a replication of the logic under `caes 'title':`, correct? However, I find it strange to use `hasOwnProperty` for this, did you just use this from the other case?